src/statmodels.jl

```diff
@@ -39,6 +39,35 @@ the likelihood of the model.
 """
 deviance(obj::StatisticalModel) = error("deviance is not defined for $(typeof(obj)).")
 
+"""
+    hessian(obj::StatisticalModel)
+
+Return the Hessian of the statistical model.
+"""
+hessian(obj::StatisticalModel) = error("hessian is not defined for $(typeof(obj)).")
+
+"""
+    score(obj::StatisticalModel)
+
+Return the score of the statistical model. The score is the gradient of the
+log-likelihood with respect to the coefficients.
+"""
+score(obj::StatisticalModel) = error("score is not defined for $(typeof(obj)).")
+
+"""
+    leverage(obj::StatisticalModel)
+
+Returns the diagonal of the projection matrix.
+"""
+leverage(obj::StatisticalModel) = error("leverage is not defined for $(typeof(obj)).")
+
+"""
+    linear(obj::StatisticalModel)
+
+Return Boolean indicator whether model is linear.
+"""
+linear(obj::StatisticalModel) = error("linear is not defined for $(typeof(obj)).")
```

Comment:
`islinear`?

Reply:
I originally had `islinear`, but don't have a preference between the two. We can go with `islinear`.